aspnetcore/blazor/hybrid/security/index.md

```diff
@@ -0,0 +1,74 @@
+
+## Keep `BlazorWebView` (`WebView`) current in deployed apps
+
+By default, the [`BlazorWebView`](/maui/user-interface/controls/blazorwebview) control uses the currently-installed, platform-specific native `WebView`. Since the native `WebView` is periodically updated with support for new APIs and fixes for security issues, it may be necessary to ensure that an app is using a `WebView` version that meets the app's requirements.
+
+To keep the `WebView` current in deployed apps, check the `WebView` version and prompt the user to take any necessary steps to update it.
+
+<!-- HOLD FOR RC2 AND SWAP FOR THE PRIOR SENTENCE
+
+Use one of the following approaches to keep the `WebView` current in deployed apps:
+
+* **On all platforms**: Check the `WebView` version and prompt the user to take any necessary steps to update it.
+* **Only on Windows**: Package a fixed-version `WebView` within the app, using it in place of the system's shared `WebView`.
+
+-->
+
+### Android
+
+The Android `WebView` is distributed and updated via the [Google Play Store](https://play.google.com/store/apps/details?id=com.google.android.webview). Check the `WebView` version by reading the [`User-Agent`](https://developer.mozilla.org/docs/Web/HTTP/Headers/User-Agent) string. Read the `WebView`'s [`navigator.userAgent`](https://developer.mozilla.org/docs/Web/API/Navigator/userAgent) property using [JavaScript interop](xref:blazor/js-interop/index) and optionally cache the value using a singleton service if the user agent string is required outside of a Razor component context.
+
+### iOS/Mac Catalyst
+
+iOS and Mac Catalyst both use [`WKWebView`](https://developer.apple.com/documentation/webkit/wkwebview), a Safari-based control, which is updated by the operating system. Similar to the [Android](#android) case, determine the `WebView` version by reading the `WebView`'s [`User-Agent`](https://developer.mozilla.org/docs/Web/HTTP/Headers/User-Agent) string.
+
+### Windows (.NET MAUI, WPF, WinForms)
+
+On Windows, the Chromium-based [Microsoft Edge `WebView2`](/microsoft-edge/webview2/) is required to run Blazor web apps. By default, the newest installed version of `WebView2` (known as the [Evergreen distribution](/microsoft-edge/webview2/concepts/distribution#details-about-the-fixed-version-runtime-distribution-mode)) is used.
+
+<!-- AT RC2, ADD THE FOLLOWING SENTENCE TO THE PRECEDING PARAGRAPH
+
+If you wish to ship a specific version of `WebView2` with the app, use the [Fixed Version distribution mode](/microsoft-edge/webview2/concepts/distribution#details-about-the-fixed-version-runtime-distribution-mode).
+
+-->
+
+For more information on selecting a `WebView2` distribution mode and checking the currently-installed `WebView2` version, see the [`WebView2` distribution docs](/microsoft-edge/webview2/concepts/distribution).
```

Comment:
```suggestion
For more information on checking the currently-installed `WebView2` version, see the [`WebView2` distribution docs](/microsoft-edge/webview2/concepts/distribution).
```

The removed part here is only relevant with the other additions happening at RC2.